RefRed/metadata/metadata_finder.py

```diff
@@ -398,28 +418,33 @@ def getNewListMetadataSelected(self):
     def saveMetadataListAsAsciiFile(self):
         if self.list_runs == []:
             return
-        _filter = 'List Metadata (*_metadata.txt);;All(*.*)'
+        _filter = "List Metadata (*_metadata.txt);;All(*.*)"
         _run_number = str(self.list_runs[0])
-        _default_name = self.parent.path_ascii + '/' + _run_number + '_metadata.txt'
+        _default_name = self.parent.path_ascii + "/" + _run_number + "_metadata.txt"
 
-        rst = QFileDialog.getSaveFileName(self, 'Save Metadata into ASCII', _default_name, filter=_filter)
+        rst = QFileDialog.getSaveFileName(
+            self, "Save Metadata into ASCII", _default_name, filter=_filter
+        )
         if isinstance(rst, tuple):
             filename, _ = rst
         else:
             filename = rst
 
-        if filename == '':
+        if filename == "":
             return
 
         self.parent.path_config = os.path.dirname(filename)
-        text = ['# Metadata Selected for run ' + _run_number]
-        text.append('#Name - Value - Units')
+        text = ["# Metadata Selected for run " + _run_number]
+        text.append("#Name - Value - Units")
 
         _metadata_table = self.ui.metadataTable
         nbr_row = _metadata_table.rowCount()
         for r in range(nbr_row):
-            _line = _metadata_table.item(r, 0).text() + ' ' + str(_metadata_table.item(r, 1).text()) + ' '\
-                + str(_metadata_table.item(r, 2).text())
+            _line = " ".join(
+                str(_metadata_table.item(r, 0).text()),
+                str(_metadata_table.item(r, 1).text()),
+                str(_metadata_table.item(r, 2).text()),
+            )
```

Comment:
`str.join` takes a single iterable (usually a list), so these will lead to an error